swap/cheque.go

```diff
@@ -131,7 +131,7 @@ func (cheque *Cheque) verifyChequeProperties(p *Peer, expectedBeneficiary common
 		return fmt.Errorf("wrong cheque parameters: expected contract: %x, was: %x", p.contractAddress, cheque.Contract)
 	}
 
-	// the beneficiary is the owner of the counterparty swap contract
+	// the beneficiary is the issuer of the counterparty swap contract
```

Comment:
since a contract is not issued (unlike a cheque) i don't think the terminology change should be applied in this case

Reply:
the contract does not use the `owner` terminology anymore. But you are correct that the contract is not issued (the "issuer" is the party allowed to issue cheques but not the "issuer" of the contract). `allowed issuer` or `issuing party` might be some option, but I think outside the context of a cheque we can also keep it as `owner` on the go side.

Reply:
I will commit the changed comment, `the beneficiary is the issuer of cheques of the counterparty swap contract`.

I replaced all instances just to be on the safe side, but I agree it depends on the context.
I agree that it's a bit confusing.